WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteSettingsViewController+SiteManagement.swift

```diff
@@ -87,7 +88,9 @@ public extension SiteSettingsViewController {
 
                 if purchases.isEmpty {
                     WPAppAnalytics.track(.siteSettingsDeleteSiteAccessed, with: strongSelf.blog)
-                    strongSelf.present(strongSelf.confirmDeleteController(), animated: true, completion: nil)
+                    let deleteSiteVC = UIStoryboard(name: "DeleteSite", bundle: nil).instantiateViewController(withIdentifier: "DeleteSiteViewController") as! DeleteSiteViewController
```

Comment:
Nitpick, but the storyboard and view controller names could be static constants in `DeleteSiteViewController` (or `DeleteSiteViewController` could contain a class method `controller(blog:)` and instantiate itself from the storyboard). e.g. the `controller` method in `SigninSelfHostedViewController`.